app/src/main/java/com/antosik/benchproject/app/common/binding/view/ViewBindingAdapters.kt

```diff
@@ -2,8 +2,18 @@ package com.antosik.benchproject.app.common.binding.view
 
 import android.view.View
 import androidx.databinding.BindingAdapter
+import com.antosik.benchproject.app.common.state.UIState
 
 @BindingAdapter("isVisible")
 fun View.setVisibility(flag: Boolean) {
     visibility = if (flag) View.VISIBLE else View.GONE
 }
+@BindingAdapter("showOnLoading")
+fun View.showOnLoading(uiState: UIState) {
```

Comment:
Binding adapters should be unit tested if it's possible

Reply:
Ok :D